### PR TITLE
Make vtexignore consistent with gitignore

### PR DESCRIPTION
**Pull request by victorcolombo**
#### What is the purpose of this pull request?
* `.vtexignore` now follows the syntax of [.gitignore](https://git-scm.com/docs/gitignore)
* `.vtexignore` contains implicitly all the rules of `.gitignore`, making it consistent with `ci-hub` 

#### What problem is this solving?
Current behavior is not intuitive.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (marcoskwkm) [inline CHANGELOG.md]: Don't forget the changelog =)

File: src/modules/apps/file.ts [hunk 2]
```diff
@@ -110,17 +91,9 @@ export async function createLinkConfig(appSrc: string) : Promise<LinkConfig> {
...
-    'README.md',
-    '.eslintrc',
-    '.gitignore',
-    'CHANGELOG.md',
-    'node_modules/**',
-    '**/node_modules/**',
-  ]
   const getFiles = async ([module, path]) => {
-    return await glob(['**'], { cwd: path, ignore, nodir: true, })
+    console.log('CHAMEI')
```
Comment (marcoskwkm) [inline src/modules/apps/file.ts]: Remove.

File: src/modules/apps/file.ts [hunk 3]
```diff
@@ -142,44 +115,35 @@ export function getLinkedDepsDirs(linkConfig : LinkConfig): string[] {
...
+  const ignoreFiles = ['.vtexignore', '.gitignore']
+  const ignoredPatterns = await Promise.all(map(async file => {
+    try {
+      const content = await readFile(join(root, file))
+      return content.toString().split('\n')
+    } catch (e) {
+      return []
+    }
+  }, ignoreFiles))
+  return ignore().add(reduce((x, y) => [...x, ...y], [], ignoredPatterns))
```
Comment (marcoskwkm) [inline src/modules/apps/file.ts]: I think you can use Ramda's `flatten` method here instead of `reduce`ing the array.

**Comment by tiagonapoli** [comment]
I’m closing this pull request _for now_ because of inactivity.

**Pull request closed without merging**